loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -484,24 +485,19 @@ static void binary_update_new_version_test(void)
 		return;
 	}
 
-	ret = binary_update_reload(APP_NAME);
+	ret = binary_update_reload(bin_name);
 	if (ret != OK) {
 		return;
 	}
 
-	sleep(2);
+	sleep(3);
 
-	ret = binary_update_getinfo(APP_NAME, &cur_bin_info);
+	ret = binary_update_getinfo(bin_name, &cur_bin_info);
 	if (ret != OK) {
 		return;
 	}
 
-	ret = binary_update_check_test_result(&pre_bin_info, &cur_bin_info, DOWNLOAD_VALID_BIN);
-	if (ret == OK) {
-		/* Unlink binary file with old version */
-		snprintf(filepath, BINARY_PATH_LEN, "%s/%s_%u", BINARY_DIR_PATH, APP_NAME, (uint32_t)pre_bin_info.version);
-		unlink(filepath);
-	}
+	binary_update_check_test_result(&pre_bin_info, &cur_bin_info, DOWNLOAD_VALID_BIN);
```

Comment:
In previous, old version binary was unlinked. But in here, you removed the unlink.
Who will unlink the old one? Is it not needed?